Rest/Controller/ContentController.php

```diff
@@ -278,4 +292,53 @@ private function getRelation($content, $language)
 
         return false;
     }
+
+    /**
+     * Returns author of the content.
+     *
+     * @param Content $contentValue
+     * @param ContentType $contentType
+     *
+     * @return string
+     */
+    private function getAuthor(Content $contentValue, ContentType $contentType)
+    {
+        $author = $contentValue->getFieldValue($this->getFieldIdentifier('author', $contentType));
+
+        if (null === $author) {
+            $user = $this->userService->loadUser($contentValue->contentInfo->ownerId);
```

Comment:
I didn't know that user can be treated like other content values. Fixed, thanks.